README.md

```diff
@@ -76,6 +76,26 @@ make test
 
 You need `docker-compose` to run the integration tests.
 
+## Running the integration locally with a vCenter simulator
+
+For testing purposes there is the possibility to run a docker compose cluster with 2 containers:
+- One NewRelic Infrastructure agent with the integration installed.
+- One container with [VCSIM](https://github.com/vmware/govmomi/tree/master/vcsim) running with the defaults to simulate a VCenter in port 8989.
```

Comment:
maybe a small comment saying that we can modify # of hosts, vms, etc (burning your mac down)

Reply:
I have added the possibility to set the Hosts, Vms and DSs  by env vars.

Reply:
done